editor/plugins/text_shader_editor.cpp

```diff
@@ -383,11 +383,12 @@ void ShaderTextEditor::_code_complete_script(const String &p_code, List<ScriptLa
 	List<ScriptLanguage::CodeCompletionOption> pp_defines;
 	ShaderPreprocessor preprocessor;
 	String code;
-	complete_from_path = (shader.is_valid() ? shader->get_path() : shader_inc->get_path()).get_base_dir();
+	String resource_path = (shader.is_valid() ? shader->get_path() : shader_inc->get_path());
```

Comment:
The parentheses are now redundant, but it's probably not worth amending the commit just for it. Let's wait to see if reviewers have other things they want updated.